### PR TITLE
Do not build stdlib/bigint.c while it cannot be built with llvm 14

### PR DESCRIPTION
**Pull request by seanyoung**
This breaks the container build.

Signed-off-by: Sean Young <sean@mess.org>

### CODE REVIEW

File: integration/substrate/asserts.spec.ts [hunk 1]
```diff
@@ -29,7 +29,7 @@ describe('Deploy asserts contract and test', () => {
         expect(res0.output?.toJSON()).toEqual(1);
 
         let res1 = await query(conn, alice, contract, "testAssertRpc");
-        expect(res1.result.toHuman()).toEqual({ "Err": { "Module": { "error": "0x0b000000", "index": "7" } } });
+        expect(res1.result.toHuman()).toEqual({ "Err": { "Module": { "error": "0x0b000000", "index": "8" } } });
```
Comment (LucasSte) [inline integration/substrate/asserts.spec.ts:32]: Why did you change this file?
It seems unrelated to this PR.

Comment (seanyoung) [reply integration/substrate/asserts.spec.ts:32]: The substrate tests don't work anymore unless I change this.

Comment (seanyoung) [reply integration/substrate/asserts.spec.ts:32]: @xermicus do you know anything about this?

Comment (xermicus) [reply integration/substrate/asserts.spec.ts:32]: Our CI uses some  `latest` docker image for the substrate contracts node. There was an update published a couple hours ago which is causing the error.

Before the update, the `contracts` pallet was the 7th module in the pallet, but now it is the 8th. The but the actual error is still the same (`ContractTrapped`, which is granted, not very obvious here), so this change is fine. Unless you want me to open a dedicated PR to fix this

**Review by xermicus — APPROVED** [review]

File: integration/substrate/asserts.spec.ts (hunk 1 above)
Comment (xermicus) [reply integration/substrate/asserts.spec.ts:32]: #1146 will fix this, sorry

**Review by LucasSte — APPROVED** [review]
